testing/simapp/app.go

```diff
@@ -117,8 +117,8 @@ import (
 	ibcfee "github.com/cosmos/ibc-go/v8/modules/apps/29-fee"
 	ibcfeekeeper "github.com/cosmos/ibc-go/v8/modules/apps/29-fee/keeper"
 	ibcfeetypes "github.com/cosmos/ibc-go/v8/modules/apps/29-fee/types"
-	transfer "github.com/cosmos/ibc-go/v8/modules/apps/transfer"
-	ibctransferkeeper "github.com/cosmos/ibc-go/v8/modules/apps/transfer/keeper"
+	"github.com/cosmos/ibc-go/v8/modules/apps/transfer"
+	ibctransferkeeper "github.com/cosmos/ibc-go/v8/modules/apps/transfer/keeper" //nolint
```

Comment:
This is an interesting one that left me stumped!

If I remove the type alias, `goimports` will remove the import saying it is unused (?!) and it won't build)
If I replace it with `_`, the code will not build because `ibctransferkeeper` is not found 
If I don't do anything, the linter will complain ¯\_(ツ)_/¯

So I would be very happy if anyone would like to help me understand what is going on :)

Reply:
Well, this is weird. I tried to repro the issue (which happened at line 121) but... it's not happening anymore. No idea what was happening, but thank you!